deploy/docker-ephemeral/build/README.md

```diff
@@ -0,0 +1,54 @@
+A makefile that uses docker.io, and qemu-user-static to build dependencies for our integration tests.
+
+Builds and uploadsdocker images for multiple architectures. Allows for '-j' to build multiple images at once. Uploads assume the hub.docker.com docker registry.
+
+# Setup
+
+## Docker
+
+Follow the instructions in [our dependencies file](doc/Dependencies.md) to ensure you have docker installed, and logged in.
+
+## qemu
+
+### Debian
+
+
+```bash
+apt-get install qemu-user-static
+sudo service binfmt-support start
+```
+
+### Fedora
+
+'sudo dnf install -y qemu-user-static'
+
+# Using
+
+Assuming you have docker, and have followed the above instructions, "make build-all" should work. This builds all of the images, and places them in docker on the local machine.
+to build an individual image (and it's dependent images), run "make-<imagename>". to see a list of images that are buildable, run "make names".
+
+## Using with Dockerhub
+
+If you want to upload images to dockerhub, you must go to dockerhub, and create repositories under your user with the names of the images you want to upload. Again, to get the list of names buildable with this Makefile, type 'make names'.
+
+If you don't want to change the Makefile, add the DOCKER_USERNAME, DOCKER_EMAIL, and DOCKER_REALNAME environment variables.
+
+For instance, when I want to build all debian images, and upload them to dockerhub, i use:
+```bash
+make DIST=DEBIAN DOCKER_USERNAME=julialongtin DOCKER_EMAIL=julia.longtin@wire.com DOCKER_REALNAME='Julia Longtin' push-all
+```
+
+You can also push a single image (and it's dependencies) with "make push-<imagename>".
```

Comment:
Should the makefile have this functionality? Kinda seems like feature-bloat; what does "it's dependencies" actually mean?

Reply:
request from jschaul. as for dependencies, we have to build java_maven_node_python for localstack, and we build airdock_base and airdock_rvm in order to build airdock_fakesqs.